### PR TITLE
feat(snippets): hide lyrics button

### PR DESCRIPTION
**Pull request by Sakura125809**

### CODE REVIEW

**Comment by CharlieS1103** [comment]
I think this might be better suited as a toggle in the better lyrics setting menu, I would recommend making a PR implementing that yourself or making an issue requesting the maintainer of better lyrics to do so. Let me know what you think though.

**Comment by rxri** [comment]
Snippets shouldn't be extension dependent. Please make PR to betterLyrics repo

**Comment by Sakura125809** [comment]
> I think this might be better suited as a toggle in the better lyrics setting menu, I would recommend making a PR implementing that yourself or making an issue requesting the maintainer of better lyrics to do so. Let me know what you think though.

I’m very sorry, but Spotify lyrics vary greatly by region, resulting in many songs without lyrics. For me (and maybe others have the same demand), the lyrics button is often useless, so I developed this snippet to remove this button.

**Comment by Sakura125809** [comment]
> Snippets shouldn't be extension dependent. Please make PR to betterLyrics repo

As I answered CharlieS1103, I can delete the part about better lyrics (you can too, because I turned on the Allow edits by maintainers button). This snippet only removes the native lyrics button, because many songs are limited by region and have no lyrics.

**Comment by Sakura125809** [comment]
@theRealPadster

File: src/resources/snippets.ts [hunk 1]
```diff
@@ -317,4 +317,10 @@ export default [
     "code": ".lyrics-lyrics-contentWrapper { text-align: center; }",
     "preview": "resources/assets/snippets/centered-lyrics.png",
   },
+  {
+    "title": "Hide lyrics button",
+    "description": "Hides the lyrics button in the playbar",
+    "code": "button.Button-sc-1dqy6lx-0.Button-sm-16-buttonTertiary-iconOnly-useBrowserDefaultFocusStyle.main-genericButton-button:has(span):has(svg):has(path[d='M13.426 2.574a2.831 2.831 0 0 0-4.797 1.55l3.247 3.247a2.831 2.831 0 0 0 1.55-4.797zM10.5 8.118l-2.619-2.62A63303.13 63303.13 0 0 0 4.74 9.075L2.065 12.12a1.287 1.287 0 0 0 1.816 1.816l3.06-2.688 3.56-3.129zM7.12 4.094a4.331 4.331 0 1 1 4.786 4.786l-3.974 3.493-3.06 2.689a2.787 2.787 0 0 1-3.933-3.933l2.676-3.045 3.505-3.99z']) {display: none;}",
```
Comment (theRealPadster) [inline src/resources/snippets.ts:323]: Is there not a better way to select the element without hardcoding an entire SVG path in there? Seems unnecessarily long and prone to breakage if it ever changes. I don't seem to have a lyrics button in my Spotify, so I can't test it or look for a better option. Do you have have idea how I'd get the lyrics button?

**Review by theRealPadster — CHANGES_REQUESTED** [review]

File: src/resources/snippets.ts (hunk 1 above)
Comment (Sakura125809) [reply src/resources/snippets.ts:323]: Yes, I will correct this part.

Comment (Sakura125809) [reply src/resources/snippets.ts]: Now I have corrected it and it works on my device.

**Review by theRealPadster — APPROVED** [review]
Nice that looks better, thanks

**Comment by SunsetTechuila** [comment]
this snippet also hides the right sidebar button🤦‍♂️

**Comment by SunsetTechuila** [comment]
selector should be `.main-nowPlayingBar-lyricsButton`

**Comment by Sakura125809** [comment]
> selector should be `.main-nowPlayingBar-lyricsButton`

Yes, this is an oversight on my part and I will correct it immediately. thank you for pointing out. :)

**Pull request merged**